ci/appveyor.yml

```diff
@@ -0,0 +1,251 @@
+
+    source $HOME/venv3.9/bin/activate
+
+    python ci/ext.py wheel
+
+    DT_WHEEL=`ls dist/*-cp39-*.whl`
+
+    echo "----- _build_info.py for Python 3.9 ------------------------------"
+
+    cat src/datatable/_build_info.py
+
+    echo "------------------------------------------------------------------"
+
+    pip --disable-pip-version-check install $DT_WHEEL pytest docutils pandas
+
+    pytest -ra --maxfail=10 -Werror -vv --showlocals ./tests/
+
+
+# Windows builds
+- ps: >-
+
+    # =======================================================================
+    #   Set up environment variables
+    # =======================================================================
+
+    $env:DT_HARNESS = "AppVeyor"
+
+    $DEFAULT_PATH = $env:PATH
+
+    $MAIN_BUILD_ID = git rev-list --count main
+
+
+    # Do not take into account the AppVeyor's additional merge commit
+
+    $BRANCH_BUILD_ID = (git rev-list --count main) - 1
+
+    if ($env:APPVEYOR_PULL_REQUEST_NUMBER -ne $null) {
+      if ($env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH -match "^rel\-") {
+        $env:DT_RELEASE = "True"
+      } else {
+        $env:DT_BUILD_SUFFIX = "PR$env:APPVEYOR_PULL_REQUEST_NUMBER.$BRANCH_BUILD_ID"
+      }
+    }
+    elseif ($env:APPVEYOR_REPO_BRANCH -eq "main") {
+      $env:DT_BUILD_NUMBER = $MAIN_BUILD_ID
+    }
+    elseif ($env:APPVEYOR_REPO_BRANCH -match "^rel\-") {
+      $env:DT_RELEASE = "True"
+    }
+    else {
+      $env:DT_BUILD_SUFFIX = "$env:APPVEYOR_REPO_BRANCH.$BRANCH_BUILD_ID"
+    }
+
+
+    # =======================================================================
+    #   Print debug information
+    # =======================================================================
+
+    echo "DEFAULT_PATH = $DEFAULT_PATH"
+
+    echo "APPVEYOR_REPO_BRANCH = $env:APPVEYOR_REPO_BRANCH"
+
+    echo "APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH = $env:APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH"
+
+    echo "APPVEYOR_PULL_REQUEST_NUMBER = $env:APPVEYOR_PULL_REQUEST_NUMBER"
+
+    echo "MAIN_BUILD_ID = $MAIN_BUILD_ID"
+
+    echo "BRANCH_BUILD_ID = $BRANCH_BUILD_ID"
+
+    echo "DT_RELEASE = $env:DT_RELEASE"
+
+    echo "DT_BUILD_NUMBER = $env:DT_BUILD_NUMBER"
+
+    echo "DT_BUILD_SUFFIX = $env:DT_BUILD_SUFFIX"
+
+
+
+    if ($env:DT_RELEASE -eq "True") {
+
+      # =======================================================================
+      #   Build and test wheel for Python 3.6
+      # =======================================================================
+
+      $env:PATH = "C:/Python36-x64;C:/Python36-x64/Scripts;$DEFAULT_PATH"
+
+      python ci/ext.py wheel
+
+      $DT_WHEEL = ls dist/*-cp36-*.whl
+
+      echo "----- _build_info.py for Python 3.6 ------------------------------"
+
+      cat src/datatable/_build_info.py
+
+      echo "------------------------------------------------------------------"
+
+      pip --disable-pip-version-check install $DT_WHEEL pytest docutils pandas

```

Comment:
Let's install latest version of pip instead

Reply:
I'm trying that now to see how long it takes